### DISCUSSION TITLE
💡please add feature to edit vector brush lines like Toon Boom harmony

### DISCUSSION THREAD

**Discussion by ArtistBirdy** [keiwando/toonsquid#361] [Ideas]
like this youtube tutorial:
https://youtu.be/cz88m7YdHVA

**Comment by amoraleite** [comment]
Would be great if we could adjust the line of a vector brush. Maybe something more simple, just control the points in a line it would be good! 

More one thing, guys in my iPad 9 the vector brush is too slow…

**Reply by DWrightAnimation** [reply]
I 100% endorse this idea.

**Reply by VeiloPot** [reply]
> I 100% endorse this idea.

So do I!
